.github/workflows/check-heroku-tests.yml

```diff
@@ -0,0 +1,83 @@
+name: Heroku Tests
+
+on:
+  workflow_dispatch:
+    inputs:
+      commit_hash:
+        description: 'Hash of git commit to run heroku tests on'
+        required: true
+  schedule:
+  - cron: '0 7 * * 1'  # M H d m w (Mondays at 7:00)
```

Comment:
We have the same schedule in current wf. Can we not duplicate it and just detect if WF was triggered by schedule? Also we talked about having some specific string in commit message to trigger heroku tests even in not-scheduled runs

Reply:
Doing a check on title of the PR now

Reply:
schedule or title check -- should work on both now